chirp/drivers/idrp.py

```diff
@@ -81,7 +81,7 @@ def set_freq(pipe, freq):
     resp = send(pipe, buf)
     for frame in resp:
         if len(frame) == 6:
-            if frame[4] == b"\xfb":
+            if frame[4] == 251:
```

Comment:
Ah yep, this was incorrect py2->py3 conversion. I figured it was something like this.